### PR TITLE
Add int and skipna

### PR DESCRIPTION
**Pull request by Rubtsowa**

### CODE REVIEW

**Comment by AlexanderKalistratov** [comment]
@Rubtsowa @PokhodenkoSA is this PR dropped?

**Comment by PokhodenkoSA** [comment]
> @Rubtsowa @PokhodenkoSA is this PR dropped?

I do not like the architecture of the solution. It should be discussed further.

**Comment by AlexanderKalistratov** [comment]
> > @Rubtsowa @PokhodenkoSA is this PR dropped?
> 
> I do not like the architecture of the solution. It should be discussed further.

I'm OK with it. But we need to test performance with ints too. So, let's not spent on it 2 weeks somehow

File: sdc/tests/tests_perf/data_generator.py [hunk 1]
```diff
@@ -16,14 +16,17 @@ def gen_str_data(data_num, data_length, input_data, data_width):
     return data
 
 
-def gen_data(data_num, data_length, input_data):
+def gen_data(data_num, data_length, input_data, typ):
```
Comment (densmirn) [inline sdc/tests/tests_perf/data_generator.py]: Are `input_data` and `typ` connected? If it's so, please use only one of these ones. There is a problem if `type(input_data)` doesn't match `typ`.

**Comment by AlexanderKalistratov** [comment]
any updates here?

File: sdc/tests/tests_perf/generator.py [hunk 2]
```diff
@@ -18,11 +18,11 @@ class TestCase(NamedTuple):
     """
     name: str
     size: list
-    params: str = ''
+    params: list = ['']
```
[the remark this answers is not in the document]
Comment (Rubtsowa) [reply sdc/tests/tests_perf/generator.py]: we now transmit test data of various types, and not just 1.

[the remark this answers is not in the document]
Comment (PokhodenkoSA) [reply sdc/tests/tests_perf/generator.py]: Ok, now `params` contains data for separate cases.

**Comment by Rubtsowa** [comment]
@densmirn and @PokhodenkoSA don't like this solution.

**Pull request closed without merging**